ci/test_inference.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd /nvtabular/
+git pull origin main
+
+tritonserver --model-repository /model/models --backend-config=tensorflow,version=2 --model-control-mode=explicit &
+sleep 1m
```

Comment:
We have support here for running the inference server - and polling until its started (rather than just sleeping a fixed amount of time).
https://github.com/NVIDIA/NVTabular/blob/84ad9776e82c9be84259bd6d11ad54769bbce210/tests/unit/test_triton_inference.py#L23-L47

I'm wondering if we should move this function up one level (to tests/conftest.py) and then use in both the integration tests and unittest, instead of starting in this shell script.

Reply:
This does not apply anymore. Servers are being launched within the tests now.